test/parallel/test-fs-exists.js

```diff
@@ -30,6 +30,8 @@ fs.exists(f, common.mustCall(function(y) {
   assert.strictEqual(y, true);
 }));
 
+assert.doesNotThrow(fs.exists(f));
```

Comment:
This isn't quite right yet. In this case, `fs.exists(f)` will be evaluated, and the return value will be passed to `assert.doesNotThrow()`. If you wrap `fs.exists(f)` in a function first, then it will be good to go. It would look like:

`assert.doesNotThrow(() => { fs.exists(f); });`